pkgs/development/tools/misc/asls/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv
+, fetchurl
+, erlangR22
+}:
+
+stdenv.mkDerivation rec {
+  pname = "asls";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = "https://github.com/saulecabrera/asls/releases/download/v0.3.0/bin.tar.gz";
```

To simplify updates:
```suggestion
    url = "https://github.com/saulecabrera/asls/releases/download/v${version}/bin.tar.gz";
```

---

Good point. Fixed!

---

I also bumped the version to init at `0.4.0` instead, given that I just released this version today.